TheEthicalHackerCup/Assets/Tutorial/Scripts/TutorialTextTyper.cs

```diff
@@ -8,12 +8,20 @@ public class TutorialTextTyper : MonoBehaviour
     [SerializeField] private TextTyper tt;
     [SerializeField] private TextAsset textFile;
     private Queue<string> textQueue;
+
+    private static bool tutorialOver; 
```

Comment:
This being static might lead to some strange behavior with seeing multiple tutorials over and over again. It'd be safer to make this a regular private field and then make Tutorial.cs have a reference to it from in the Unity prefab inspector